### PR TITLE
Update dbaas terraform guide

### PR DESCRIPTION
**Pull request by mijanss**

### CODE REVIEW

File: content/en/docs/dbaas/guides/terraform_backend.md [hunk 1]
```diff
@@ -8,82 +8,90 @@ alwaysopen: true
...
+Make sure you've copied and saved the client secret before you close the dialog. The client secret value cannot be obtained later and you will have to create a new one.
+
+![Copy credential](/img/dbaas/guides/dbaas_terraform_backend-2.png)
+
+---
+
+## Terraform configuration
+
+Create a new file locally with the variables below and add your Client ID and Client Secret.
 ```json
```
Comment (nyxi) [inline content/en/docs/dbaas/guides/terraform_backend.md:43]: This codeblock contains a shell script, not JSON data. Please change highlighting to `shell`.

File: content/en/docs/dbaas/guides/terraform_backend.md [hunk 2]
```diff
@@ -8,82 +8,90 @@ alwaysopen: true
...
 Source your newly created credentials file.
 ```json
 source /path/to/myfile.sh
 ```
 
-## Terraform provider
-Create a new terraform configuration file, for instance `provider.tf` and add the CCX provider.
+### Terraform provider
+Create a new terraform configuration file. In this example we create `provider.tf` and add the CCX provider.
 ```json
```
Comment (nyxi) [inline content/en/docs/dbaas/guides/terraform_backend.md:57]: This codeblock does not contain JSON data, please change highlighting to `hcl`

File: content/en/docs/dbaas/guides/terraform_backend.md [hunk 3]
```diff
@@ -8,82 +8,90 @@ alwaysopen: true
...
     }
   }
 }
 ``` 
 
-## Create your first datastore with Terraform
-Create a new terraform configuration file, for instance `main.tf` and add a new resource with your datastore settings.
+### Create your first datastore with Terraform
+Create an additional terraform configuration file and add your prefered datastore settings. In this example we create a configuration file named `main.tf` and specify that his is a single node datastore with MariaDB.
 ```json
```
Comment (nyxi) [inline content/en/docs/dbaas/guides/terraform_backend.md:70]: This codeblock does not contain JSON data, please change highlighting to `hcl`

**Review by nyxi — CHANGES_REQUESTED** [review]
Good job! Just a few nitpicks from earlier then we are good to go

File: content/en/docs/dbaas/guides/terraform_backend.md (hunk 1 above)
Comment (mijanss) [reply content/en/docs/dbaas/guides/terraform_backend.md]: Fixed.

**Pull request merged**